src/components/AppNavigation/AppNavigationItem.vue

```diff
@@ -95,7 +95,7 @@
 </template>
 
 <script>
-import PopoverMenu from 'components/PopoverMenu/PopoverMenu'
+import PopoverMenu from '../PopoverMenu/PopoverMenu'
```

Comment:
Maybe I'm missing something but without that change, it doesn't seem to be able to resolve the module name. 

```
Module not found: Error: Can't resolve 'components/PopoverMenu/PopoverMenu' in '/home/jus/repos/nextcloud/vue-components/src/components/AppNavigation'

```

Reply:
Hum, this was supposed to be set 
https://github.com/nextcloud/vue-components/blob/f27d675871a16916e944cfbb5e1af196fd26524b/webpack.common.js#L48

Reply:
The issue is actually that my example apps webpack config doesn't include that so it cannot resolve it when importing the vue file directly. Any specific reason you prefer the alias?

Reply:
@juliushaertl readability.
You mean that the import vue-nextcloud-component doesn't work with the alias on your app, right?
Because I can compile this like that :)

Reply:
Nope, I use `import { AppNavigation } from 'vue-nextcloud-component' so webkit can just bundle the components i use and not include the whole bundled component js.

Reply:
That would be a huge benefit in my oppinion.

Reply:
But I guess I could also just add an alias for Components to the apps webpack config, let me try that.

Reply:
@juliushaertl it's strange that webpack does not automatically transforms the url?
https://github.com/vuematerial/vue-material/blob/dev/src/components/MdRadio/MdRadio.vue#L18

Reply:
Probably an issue with `npm link` then ... Fine to get this in for now from my side. Maybe we can have a look together at the conference.

Reply:
yes if that's ok. 
Otherwise we'll go back to relative path, it's not that big of a deal :)